schedule/publiccloud/mau-extratests-publiccloud.yml

```diff
@@ -17,44 +17,32 @@ schedule:
   - console/check_os_release
   - console/cleanup_qam_testrepos
   - console/openvswitch
-  - console/shar
   - console/sshd
-  - console/update_alternatives
   - console/rpm
-  - console/slp
-  - console/pkcon
-  - console/libvorbis
   - console/command_not_found
   - console/openssl_alpn
   - console/cron
   - console/syslog
   - console/mta
   - console/check_default_network_manager
-  - console/sqlite3
   - console/gdb
   - console/sysctl
   - console/sysstat
   - console/wget_ipv6
-  - console/ca_certificates_mozilla
   - console/gpg
-  - console/clamav
   - console/shells
   - console/sudo
-  - console/dstat
   - console/supportutils
   - console/journalctl
   - console/quota
   - console/rpcbind
-  - sysauth/sssd
   - console/timezone
   - console/procps
   - console/kmod
```

Comment:
what about this one? img-proof already has a test for kmod

Reply:
Removed.